src/AdScreenerTool.js

```diff
@@ -138,14 +142,16 @@ const AdScreener = (params) => {
 
   const [startDate, setStartDate] = useState((startDateParam===undefined) ? addDays(new Date(), -7) : new Date(startDateParam));
   const [endDate, setEndDate] = useState((endDateParam===undefined) ? new Date() : new Date(endDateParam));
-  const [topic, setTopic] = useState({ selectedOption: getSelectorValue(params.topics,topicParam)});
-  const [region, setRegion] = useState({ selectedOption: getSelectorValue(params.regions,regionParam)});
-  const [gender, setGender] = useState({ selectedOption: getSelectorValue(params.genders,genderParam)});
-  const [ageRange, setAgeRange] = useState({ selectedOption: getSelectorValue(params.ageRanges,ageRangeParam)});
-  const [riskScore, setRiskScore] = useState({ selectedOption: getSelectorValue(params.riskScores,riskScoreParam)});
-  const [orderBy, setOrderBy] = useState({ selectedOption: getSelectorValue(params.orderByOptions,orderByParam)});
-  const [orderDirection, setOrderDirection] = useState({ selectedOption: getSelectorValue(params.orderDirections,orderDirectionParam)});
-  
+  const [topic, setTopic] = useState({ selectedOption: getSelectorValue(params.topics,topicParam,true)});
+  const [region, setRegion] = useState({ selectedOption: getSelectorValue(params.regions,regionParam,false)});
+  const [gender, setGender] = useState({ selectedOption: getSelectorValue(params.genders,genderParam,false)});
+  const [ageRange, setAgeRange] = useState({ selectedOption: getSelectorValue(params.ageRanges,ageRangeParam,false)});
+  const [riskScore, setRiskScore] = useState({ selectedOption: getSelectorValue(params.riskScores,riskScoreParam,false)});
+  const [orderBy, setOrderBy] = useState({ selectedOption: getSelectorValue(params.orderByOptions,orderByParam,false)});
+  const [orderDirection, setOrderDirection] = useState({ selectedOption: getSelectorValue(params.orderDirections,orderDirectionParam,false)});
+  const [keyword, setKeyword] = useState(null);
+  const [key, setKey] = useState('topics');
```

Comment:
what does `key` mean in this context? is this for topic vs keyword search? if so, how about selectedTopicOrFullTextSearchTab? of something like that